docker-compose.dev.yml

```diff
@@ -42,9 +42,10 @@ services:
       - mongodb
     networks:
       - ets-dev
-
-  lambda:
-    container_name: ${PROJECT}-lambda
+  
+  ### Lambda Function #############################
```

Comment:
Total nitpick - just for consistency's sake, the other comments in this file were removed so IMO we should either remove these, too, or make them more descriptive if we want to keep them to explain what the multiple lambdas are 😄

Reply:
True!